src/components/MegaMenu/MegaMenu.vue

```diff
@@ -4,6 +4,7 @@
     data-testId="menu-container"
     @mouseenter="showNav = true"
     @mouseleave="showNav = false"
+    @keyup.enter="onClick"
```

Comment:
Beware, this works on mobile/small screens, but not on desktop/full screen.

There are two variables: `showNav` and `showMobileNav`. Classes are applied depending on these two & the width's screen.

Reply:
Ok, made changes.  Ready for re-review.